x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+
+import { EuiForm } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { EditTransformFlyoutFormTextInput } from './edit_transform_flyout_form_text_input';
+import { UseEditTransformFlyoutReturnType } from './use_edit_transform_flyout';
+
+interface EditTransformFlyoutFormProps {
+  editTransformFlyout: UseEditTransformFlyoutReturnType;
+}
+
+export const EditTransformFlyoutForm: FC<EditTransformFlyoutFormProps> = ({
+  editTransformFlyout: [state, dispatch],
+}) => {
+  const formFields = state.formFields;
+
+  return (
+    <EuiForm>
+      <EditTransformFlyoutFormTextInput
+        errorMessages={formFields.description.errorMessages}
+        value={formFields.description.value}
+        label={i18n.translate('xpack.transform.transformList.editFlyoutFormDescriptionLabel', {
+          defaultMessage: 'Description',
+        })}
+        onChange={value => dispatch({ field: 'description', value })}
+      />
+      {/*
+      <EditTransformFlyoutFormTextInput
+        defaultValue={config.dest.index}
+        label={i18n.translate('xpack.transform.transformList.editFlyoutFormDestinationIndexLabel', {
+          defaultMessage: 'Destination Index',
+        })}
+        onChange={onChangeDestinationIndexHandler}
+      />*/}
+      <EditTransformFlyoutFormTextInput
+        errorMessages={formFields.frequency.errorMessages}
+        value={formFields.frequency.value}
+        helpText={i18n.translate('xpack.transform.transformList.editFlyoutFormFrequencyHelptext', {
+          defaultMessage:
+            'The interval between checks for changes in the source indices when the transform is running continuously. Also determines the retry interval in the event of transient failures while the transform is searching or indexing. The minimum value is 1s and the maximum is 1h. The default value is 1m.',
```

Comment:
Too much information? Remove the middle sentence - `Also determines the retry interval in the event of transient failures while the transform is searching or indexing.` ?

Reply:
It's quite a bit of text but I left it in intentionally since it came up in one of the previous discussions around that feature, it's otherwise a bit of hidden and not obvious side-effect.